### PR TITLE
Added argos 0.3.1 plus its dependencies (cmlib and pgcolorbar)

### PR DESCRIPTION
**Pull request by titusjan**
This PR is for Argos 0.3.1 and its two dependencies (cmlib and pgcolorbar).

I created the meta.yaml files with grayskull and saw no reason to modify them after reading the documentation. However, I'm new to conda forge so I may have missed something. Please let me know if I have to make modifications.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by titusjan** [comment]
Closing and reopening the issue to force executing the tests

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
Recipes look reasonable. I think the Linux build might get fixed with some help from [this](https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl)?

**Comment by titusjan** [comment]
Thanks for helping me @dopplershift . Your link put me on the right track. It says:

> If you need a fully functional binary in the test phase, you have to also provide the shared libraries via yum_requirements.txt

This was the issue for the Linux build, which works now.

File: recipes/cmlib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cmlib-{{ version }}.zip
+  sha256: 61941601a7223940c736bb65e03a04ff503d5099e22aaf03e7d619ff60706ec8
+
+build:
+  number: 0
+  noarch: python
```
[the remark this answers is not in the document]
Comment (titusjan) [reply recipes/cmlib/meta.yaml:15]: Thanks again for pointing me in the right direction. I added `entry_points` to the `meta.yaml` file. This resolved the issue.

[the remark this answers is not in the document]
Comment (dopplershift) [reply recipes/cmlib/meta.yaml:15]: Interesting. Ok.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**